components/tooltip/index.css

```diff
@@ -15,6 +15,7 @@ governing permissions and limitations under the License.
 
 .spectrum-Tooltip {
 	--spectrum-tooltip-animation-duration: var(--spectrum-animation-duration-100);
+	--spectrum-tooltip-animation-distance: var(--spectrum-spacing-75);
```

Comment:
Design spec says `spacing-75`, I'll double-check with design and revert if needed but it doesn't sound like it differs for the large platform size.